haskell/stack.yaml

```diff
@@ -13,7 +13,7 @@ extra-deps:
     https://gitlab.com/morley-framework/morley.git
     # ^ CI cannot use ssh, so we use http clone here
   commit:
-    9e69d52741e456bd0976a1ec4c45cc4c43a11722 # master, morley-1.11.1
+    56d74ababeda9aba8e194783fc5e395fa7a6b69a # morley-1.12.0
```

Comment:
I don't know if we still support `tezos-client-7.5` after this change, but in https://github.com/tqtezos/stablecoin/blob/master/haskell/README.md we have
> tezos-client executable must be available and is assumed to be in $PATH by default.

and I think it should be updated to require 8.0 or newer

Reply:
Updated README